autogen/main/variables.tf.tmpl

```diff
@@ -574,3 +574,15 @@ variable "impersonate_service_account" {
   description = "An optional service account to impersonate for gcloud commands. If this service account is not specified, the module will use Application Default Credentials."
   default     = ""
 }
+
+variable "enable_notification_config" {
```

Comment:
variables will also need to be gated

Reply:
I also put variables into beta conditional blocks.